qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -1439,9 +1439,9 @@ def test_generate_files(self):
         fp_count = qdb.util.get_count("qiita.filepath")
         self.tester.generate_files()
         obs = qdb.util.get_count("qiita.filepath")
-        # We just make sure that the count has been increased by 2, since
+        # We just make sure that the count has been increased by 6, since
         # the contents of the files have been tested elsewhere.
-        self.assertEqual(obs, fp_count + 3)
+        self.assertEqual(obs, fp_count + 5)
```

Comment:
The changes here is because with the new template there are 2 extra files generated correct?